model/project.go

```diff
@@ -294,6 +295,27 @@ type ParameterInfo struct {
 	Description     string `yaml:"description" bson:"description"`
 }
 
+type Container struct {
+	Name        string                  `yaml:"name,omitempty" bson:"name"`
```

Comment:
I'd just noticed all fields in project_parser.go or project.go had omitempty on them even if they were seemingly required fields, so I followed that pattern in case it was required. But I don't think it would cause a problem and is better to specify what fields are required and which aren't, so I'll update